### PR TITLE
Clarify status of documented extensions

### LINKED ISSUE #993: Extension attributes: specified and unspecified
While working on introducing the term *core attributes* within the specification and primer, it looks like we also have another area of ambiguity, this time with extension attributes.

Any given context attribute might be:

- Core, one of:
  - Required
  - Optional
- Extension, one of:
  - Defined in this repo
  - Not standardized at all

While we're trying to come up with consistent terms, do we want to have a consistent way of distinguishing between "arbitrary application-specific extension attributes" and "extension attributes which are specified in this repo"?

### PR DESCRIPTION
**Pull request by jskeet**
Further debate welcome; there's a balance about how much we want to say here.

Closes #993.

Signed-off-by: Jon Skeet <jonskeet@google.com>

### CODE REVIEW

File: cloudevents/primer.md [hunk 1]
```diff
@@ -367,10 +367,13 @@ majority of the cases.
...
 those two categories but would still benefit from the level of interoperability
-that comes from being well-defined, then they will be placed into the
-"extensions" category and put into
-[documented extensions](documented-extensions.md). The specification defines how
-these extension attributes will appear within a CloudEvent.
+that comes from being well-defined, this repo's
+[documented extensions](documented-extensions.md) provides a shared collaboration
+space. Each extension specification defines how it will appear within a CloudEvent.
+Extensions which are documented in this way have no special status,
+and may take breaking changes (including being removed entirely). They are not
```
Comment (duglin) [inline cloudevents/primer.md:374]: s/take/make/

**Comment by duglin** [comment]
LGTM

File: cloudevents/primer.md (hunk 1 above)
Comment (jskeet) [reply cloudevents/primer.md:374]: The extension itself doesn't make the breaking change - it "takes" it from an author etc. Basically, this isn't a typo, but I'm happy to discuss completely rewording it.

Comment (duglin) [reply cloudevents/primer.md:374]: I won't argue with an Englishman about English :-)

**Comment by duglin** [comment]
Approved on the 1/26 call

**Pull request merged**